trulens_eval/examples/experimental/virtual_example.ipynb

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    rec1 = VirtualRecord(
</pre><p>Somewhat trivial, but can you show this given a dataframe of [prompt, context, response]?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>